js/components/avatar.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <a class="avatar" :href="user.url" :title="user | display">
+      <img v-if="user.avatar" :src="user.avatar"
+          class="avatar" :alt="user | display" width="52" height="52">
```

Comment:
Sorry, I wans't clear :/
Instead of `width="52" height="52"`, I think it will be easier to maintain with `:width="size" :height="size"`
and the following:
```javascript
export cont DEFAULT_SIZE = 52;

export default {
    props: {
        user: Object,
        size: {
            type: Number,
            default: DEFAULT_SIZE
        }
    }
}
```